src/Standard/TwigCsFixer.php

```diff
@@ -28,6 +29,7 @@ public function getRules(): array
             new SingleQuoteRule(),
             new TrailingCommaSingleLineRule(),
             new TrailingSpaceRule(),
+            new IncludeFunctionRule(),
```

Comment:
I try to order rules alphabetically, can you do it too ?

Also, you will need to update the tests.

Reply:
I will do for sure, I wanted to know if its ok for you first